### PR TITLE
don't install together with javabridge package

### PR DESCRIPTION
**Pull request by raivivek**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by raivivek** [comment]
@conda-forge-admin, please rerender

**Comment by raivivek** [comment]
@ocefpaf I believe this is what you meant. I followed the docs. Let me know if I've got anything wrong.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,6 +29,8 @@ requirements:
     - python
     - {{ pin_compatible('numpy') }}
     - openjdk
+  run_constrained:
+    - javabridge ==9999999999
```
Comment (ocefpaf) [inline recipe/meta.yaml:34]: Looks correct. To be honest I only did this once, so I'll give @jakirkham a chance to look at it before you merge.

**Review by ocefpaf — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml:34]: Yeah seems reasonable to me too :)

**Comment by jakirkham** [comment]
We might want to do something similar to `javabridge` as well

**Comment by raivivek** [comment]
@jakirkham Sure — taking care of that here https://github.com/conda-forge/javabridge-feedstock/pull/5

**Pull request merged**